src/OpenIDConnectServer.php

```diff
@@ -55,6 +55,9 @@ public function __construct( string $public_key, string $private_key, array $cli
 		$this->router->add_route( '.well-known/jwks.json', new WebKeySetsHandler( $this->public_key ) );
 		$this->router->add_route( '.well-known/openid-configuration', new ConfigurationHandler() );
 		add_action( 'login_form_openid-authenticate', array( $this, 'authenticate_handler' ) );
+
+		// Cleanup.
+		$this->setup_cleanup_routine();
```

Comment:
I think this could also be moved to `src/Storage/AuthorizationCodeStorage.php`

Reply:
I realised this function is just setting up cron hooks and all cleaning related code is already in `AuthorizationCodeStorage.php` so I renamed this method to reflect that.